strawberryfields/circuitspecs/chip0.py

```diff
@@ -54,9 +54,6 @@ class Chip0Specs(CircuitSpecs):
         Rgate({phase}) | 2
         Rgate({phase}) | 3
 
-        MeasureFock() | [0]
-        MeasureFock() | [1]
-        MeasureFock() | [2]
-        MeasureFock() | [3]
+        MeasureFock() | [0, 1, 2, 3]
```

Comment:
Is this just beautification, or did it not work before? `TFBackend.measure_fock()` requires that multiple modes are always given in ascending order, does GaussianBackend too? FockBackend doesn't.

Reply:
Without using a single MeasureFock statement, this will return independent samples for each mode, rather than joint statistics

Reply:
So this is more an issue with how PNR measurements work in the gaussian backend, I suppose

Reply:
yep exactly

Reply:
For `shots==1` the effect should be exactly the same. When `shots>1`, it'd be surprising if the behavior was different. I think this is a bug in the gaussian backend.

An easy fix would be to implement the block we were talking about last week, i.e. no additional operations allowed after a multi-shot measurement.

Reply:
It's not a bug in Gaussian backend, more of a limitation that we have to accommodate. Gaussian backend doesn't update the conditional state, so subsequent calls to `measure_fock` on remaining wires are sampling from the wrong distributions (i.e., local statistics instead of joint). Effect is independent of number of shots.